common/scala/src/main/scala/whisk/connector/kafka/KafkaProducerConnector.scala

```diff
@@ -87,6 +87,9 @@ class KafkaProducerConnector(kafkahosts: String,
     val props = new Properties
     props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkahosts)
     props.put(ProducerConfig.ACKS_CONFIG, 1.toString)
+    // Allow messages with up to 5 MiB size
+    // The target topic must support this message size as well because it's larger as the default
+    props.put(ProducerConfig.MAX_REQUEST_SIZE_CONFIG, 5242880.toString)
```

Comment:
Should the 5M also be a setting here from pure config ?

Reply:
Presumably, yes. Need to thing how to cleverly thread that through 👍